test/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const assert = require('assert');
+
+process.on("unhandledRejection", err => {
+  throw err;
+});
+
+// We try to import the generated module
+const features = require(__dirname+'/../dist/cjs');
+
+const isBoolean = function (value) {
+  return value === true || value === false;
+}
+
+const checkFeature = function (featureName) {
+  const feature = features[featureName];
+  assert(feature, `The feature ${featureName} doesn't exist`);
+  feature().then(function(result) {
+    console.log(`The feature ${featureName} returned: ${result}`);
+    assert(isBoolean(result), `The feature ${featureName} returned: ${result}`);
+  })
+}
+
+checkFeature("bulkMemory");
```

Comment:
Can you iterate over the exposed functions of `features` here? I want to avoid having to check in each PR that an appropriate test was added.

Reply:
Also I think I'd prefer an explicit `async function run()` here, so we use a for-loop with explicit `await`.

Reply:
As a testing practice, is always good to have the contracts with the user/developer written and explicit rather than implicit.

If I iterate over the fields of the features, if a feature is deleted it will go unnoticed (or if all the features are deleted) as the tests will pass when they shouldn't.
In general any breaking change on the APIs should require the tests to be fixed :)